commit c44c21badb0a87e3a1aaf4bf7ec382fdc5337342
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Feb 17 09:47:11 2017 +0000

    BCM270X_DT: Add SMSC ethernet controller to DT
    
    With an ethernet node in the DT, a suitable firmware can populate the
    local-mac-address property, removing the need for a downstream patch
    to the driver to read its MAC address from a module parameter.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts b/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
index 51f575e5d201..08bf838fab55 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b-plus.dts
@@ -1,6 +1,7 @@
 /dts-v1/;
 
 #include "bcm2708.dtsi"
+#include "bcm283x-rpi-smsc9514.dtsi"
 
 / {
 	model = "Raspberry Pi Model B+";
diff --git a/arch/arm/boot/dts/bcm2708-rpi-b.dts b/arch/arm/boot/dts/bcm2708-rpi-b.dts
index 028ef91a6c4f..4e6b4dd6a8d9 100644
--- a/arch/arm/boot/dts/bcm2708-rpi-b.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-b.dts
@@ -1,6 +1,7 @@
 /dts-v1/;
 
 #include "bcm2708.dtsi"
+#include "bcm283x-rpi-smsc9512.dtsi"
 
 / {
 	model = "Raspberry Pi Model B";
diff --git a/arch/arm/boot/dts/bcm2709-rpi-2-b.dts b/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
index a4ffeff9fda6..2dc0e1204e63 100644
--- a/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
+++ b/arch/arm/boot/dts/bcm2709-rpi-2-b.dts
@@ -1,6 +1,7 @@
 /dts-v1/;
 
 #include "bcm2709.dtsi"
+#include "bcm283x-rpi-smsc9514.dtsi"
 
 / {
 	model = "Raspberry Pi 2 Model B";
diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
index d29ba72de727..781018494416 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -5,6 +5,7 @@
 #endif
 
 #include "bcm2710.dtsi"
+#include "bcm283x-rpi-smsc9514.dtsi"
 
 / {
 	model = "Raspberry Pi 3 Model B";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1182_c44c21badb0a87e3a1aaf4bf7ec382fdc5337342.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1182_c44c21badb0a87e3a1aaf4bf7ec382fdc5337342.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
